support-data-rest/src/test/java/org/uniprot/api/support/data/configure/service/ProteomeConfigureServiceTest.java

```diff
@@ -0,0 +1,36 @@
+package org.uniprot.api.support.data.configure.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.uniprot.api.support.data.configure.domain.AdvancedSearchTerm;
+import org.uniprot.api.support.data.configure.domain.UniProtReturnField;
+
+/**
+ * @author lgonzales
+ * @since 27/05/2020
+ */
+class ProteomeConfigureServiceTest {
+
+    @Test
+    void getResultFields() {
+        ProteomeConfigureService service = new ProteomeConfigureService();
+        List<UniProtReturnField> resultGroups = service.getResultFields();
+
+        assertNotNull(resultGroups);
+        assertEquals(2, resultGroups.size());
+
```

Comment:
General comment applicable to all configure service test. Is asserting count enough? Should we not assert the actual values? If it is covered somewhere else then no need to do anything.

Reply:
yes, we have unit tests that covers it..